src/card.tsx

```diff
@@ -448,10 +459,9 @@ const CardContent: React.FC<CardContentProps> = ({
                         >
                             {description}
                         </Text2>
-                    )}
-                </Stack>
-
-                {extra && <div>{extra}</div>}
+                    </div>
+                )}
+                {extra && <Box ref={extraRef}>{extra}</Box>}
```

Comment:
why did you change the `div` with a `Box`?

Reply:
Consistency. There were a few cards with Box and others with div. Do you see any reason to use div instead?